CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ merge to master and move them to a respective version upon release.
 
 ## Unreleased
 
+- Adds support for RSA-OAEP ([#95](https://github.com/airsidemobile/JOSESwift/pull/95), [#142](https://github.com/airsidemobile/JOSESwift/pull/142))
```

Comment:
Now that we have a changelog, why not shout-out the contributors a-la-fastlane?
e.g.
```
2.114.0 Improvements
* [deliver/frameit] frame latest iOS devices, skip uploading iPhone61 (#13774) via Dave Anderson
* [produce] allow to enable/disable `access_wifi` service (#13552) via Taimur Ayaz
* [scan] use slack action to post scan test results (#13791) via Gligor Kotushevski
* [spaceship] use default sort order for listing TestFlight testers (#13778) via Theodore Dubois
* [spaceship] fix creating and revoking Apple Keys (#13697) via Dominik Sokal
```

Might be a nice way to thank contributors. Anyway, just an idea.

Reply:
That's a great idea, let's do that. I tracked an issue for that: #144. That way we can mention all contributors (also from old pull requests) without leaving the scope of this pull request.

Reply:
Perfect 👍